src/docs/patterns/user-auth.mdx

```diff
@@ -0,0 +1,296 @@
+        '1: Project identifications, such as project name or project logo.',
+        '2: Optionaly, you may add in a simple geometrical background image with a white gradient background.',
+        '3: "Remember me" and "password visibility" are two features that facilitate the user, but expose some security concerns.',
+        '4: Login screens often include extra options, such as "forgot password", "sign up", and "contact your organization admin".',
+    ]}
+/>
+
+**Login through Third Party**
+
+Some applications, especially the consumer-faced ones, are authorized via third-party credentials. Different platforms may have different requirements for this. Here are some commonly used platforms and their instructions:
+
+-   [Login with Amazon](https://developer.amazon.com/docs/login-with-amazon/brand.html)
+-   [Google Sign-in](https://developers.google.com/identity/sign-in/web/sign-in)
+-   [Sign in with Apple](https://developer.apple.com/design/human-interface-guidelines/sign-in-with-apple/overview/introduction/)
+-   [Facebook Login](http://developers.facebook.com/docs/facebook-login/)
+-   [Sign in with Microsoft](https://developer.microsoft.com/en-us/identity/add-sign-in-with-microsoft)
+
+## Reset Password
+
+**Forget password**
+
+The login screen for an application should also present users with a way to recover or reset their password if they have lost or forgotten it. This is typically presented as a hyperlink below the credential fields. Clicking the link will present users with a field to enter their e-mail address. They will receive an email with a hyperlink to reset their password. The workflow involves the following screens:
+
+-   Enter email
+-   Success (email has been sent)
+-   Receive an email for resetting the password
+-   Create password
+-   Success & welcome
+
+<ImageGrid
+    images={[ForgotPasswordEnterEmail, ForgotPasswordConfirmSent, ForgotPasswordNewPassword, ForgotPasswordWelcome]}
+    caption={['1: Enter email', '2: Success (Email has been sent)', '3: Create Password', '4: Success & Welcome']}
+/>
+
+**Change password**
+
+When the users are logged in, they have the option to reset their password by going through the changing password workflow. This workflow can usually be triggered from user preference settings. The workflow involves the following screens:
+
+-   Change password
+-   Success & welcome
+
+<React.Fragment>
+    <ImageGrid
+        images={[ChangePassword, ChangePasswordWelcome]}
+        caption={['1: Change password', '2: Success & welcome']}
+    />
+    <Divider />
+</React.Fragment>
+
+# Behavior
+
+## Email Communications
+
+<!-- This might deserve its own pattern. We already have something in the backlog to design for an email template. -->
+
+Email communications are frequently used throughout the user authentication workflow since they typically involve some security changes. You should never give the users option to unsubscribe from account-related email communications.
```

Comment:
Is this more about sending emails when account information changes, or about the emails associated with the Authentication / Registration workflows (i.e., sending the codes, invites, etc.)?

Reply:
I was trying to emphasize that for the sake of the account security, so in that regard, it is closer to the first point you made. Yet it is so frequently mentioned throughout the steps above, so I think it is worthwhile to bring it up in this pattern

Reply:
I would probably break it into two sections (one for each type):

1. Emails are often sent as part of the authentication workflows ... blah blah blah. They include invitations to register, confirmation codes, etc.

2. Emails are also sent to confirm security-related changes to a user's account (such as changing a password, updating a profile, etc.)...blah blah blah. You should never allow users to unsubscribe from security-related email notifications.